packages/ui-extensions/docs/surfaces/admin/staticPages/admin-extensions.doc.ts

```diff
@@ -0,0 +1,179 @@
+          type: 'app',
+        },
+        {
+          name: 'Direct API access',
+          subtitle:
+            "You no longer need a backend proxy to talk to Shopify's GraphQL Admin API.",
+          url: '#directAPIAccess',
+          type: 'tool',
+        },
+        {
+          name: 'Custom Protocols',
+          subtitle:
+            "We've made it easier to construct URLs to navigate to common localtions.",
+          url: '#directAPIAccess',
+          type: 'gear',
+        },
+      ],
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'gettingStarted',
+      sectionContent:
+        'To get started, create a new app and cd into the app directory. Then generate an extension.',
+      title: 'Getting Started',
+      codeblock: {
+        title: 'Generate an extension',
+        tabs: [
+          {
+            code: './examples/getting-started.ts',
+            language: 'bash',
+            title: 'CLI',
+          },
+        ],
+      },
+    },
+    {
+      type: 'Generic',
+      title: 'Direct API access',
+      sectionContent:
+        "You can now make requests to shopify directly from your extension. You can simply use fetch! The access scopes you have in your extension will be the same as the access scopes you have in your app. If you need to make a request to a resource that requires a different access scope, you will update your app's scopes.",
+      anchorLink: 'directAPIAccess',
+      codeblock: {
+        title: 'Query Shopify data',
+        tabs: [
+          {
+            code: './examples/direct-api.tsx',
+            language: 'tsx',
+            title: 'Get Product Data',
+          },
+        ],
+      },
+    },
+    {
+      type: 'GenericAccordion',
+      title: 'Custom Protocols',
+      sectionContent:
+        "We've added some custom protocols to make it easier to construct URLs to navigate to common localtions.",
+      anchorLink: 'customProtocols',
+      accordionContent: [
+        {
+          title: 'shopify:admin',
+          description:
+            'Use the shopify protocol when you want to construct a URL with a root of the Shopify Admin.',
+          codeblock: {
+            title: 'shopify:admin',
+            tabs: [
+              {
+                title: 'Link to Product Page',
+                language: 'tsx',
+                code: '<Link to="shopify:admin/products/1234567890" />',
+              },
+              {
+                title: 'Fetch data',
+                language: 'ts',
+                code: `fetch("shopify:admin/api/graphql.json", {
+  method: "POST",
+  body: JSON.stringify(simpleProductQuery),
+});`,
+              },
+            ],
+          },
+        },
+        {
+          title: 'app:',
+          description:
+            'Use the app protocol when you want to construct a URL for your app. Shopify will handle constructing the base URL for your app. This works for both embedded and non-embedded apps.',
```

Comment:
`Use the app protocol to construct a URL for your app. Shopify will handle constructing the base URL for your app. This works for both embedded and non-embedded apps.`

Reply:
```suggestion
            'Use the app protocol to construct a URL for your app. Shopify will handle constructing the base URL for your app. This works for both embedded and non-embedded apps.',
```